src/jit/flowgraph.cpp

```diff
@@ -5855,24 +5809,13 @@ void Compiler::fgFindBasicBlocks()
         printf("Jump targets:\n");
         for (unsigned i = 0; i < info.compILCodeSize + 1; i++)
         {
-            if (jumpTarget[i] == JT_NONE)
-            {
-                continue;
-            }
-
-            anyJumpTargets = true;
-            printf("  IL_%04x", i);
-
-            if (jumpTarget[i] & JT_ADDR)
```

Comment:
Was this debug info valuable?

Reply:
It wasn't used anywhere else, so I'd say it was not very valuable.